test/Driver/parseable_output_unicode.swift

```diff
@@ -112,10 +114,10 @@
 // CHECK-NEXT: {
 // CHECK-NEXT:   "kind": "began",
 // CHECK-NEXT:   "name": "link",
-// CHECK-NEXT:   "command": "{{.*[\\/]}}ld{{(\\")?}} {{.*[\\/]}}你好-[[OUTPUT]].o {{.*}} -o {{.*[\\/]}}parseable_output_unicode.swift.tmp.out",
-// CHECK-NEXT:   "command_executable": "{{.*[\\/]}}ld{{(\\")?}}",
+// CHECK-NEXT:   "command": "{{.*[\\/](ld|clang\+\+.exe)(\\")?}} {{.*[\\/]}}你好-[[OUTPUT]].o{{(\\")?}}{{.*}}-o {{.*[\\/]}}parseable_output_unicode.swift.tmp.out{{(\\")?}}",
+// CHECK-NEXT:   "command_executable": "{{.*[\\/](ld|clang\+\+.exe)(\\")?}}",
 // CHECK-NEXT:   "command_arguments": [
-// CHECK-NEXT:     "{{.*[\\/]}}你好-[[OUTPUT]].o",
```

Comment:
What changed here such that this is no longer the next line?

Reply:
It passes `-target` which comes first.